Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,13 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Parallel RBD PVC creation hangs for new BlockPool
+
+This issue is specifically present in CephCSI `v3.4.x`, used by rook `>=v1.7.1` and occurs when multiple parallel PVCs creation requests are issued on a newly created blockpool. Follow the steps below to workaround the issue:
+* Delete the in-progress PVC create requests.
+* Restart the csi-rbdplugin-provisioner-xxx pods.
```

Comment:
won't this leave any stale  OMAP objects?

Reply:
> won't this leave any stale OMAP objects?

Yes, it will leave stale OMAP entries but IMO these are harmless and would require extra effort/steps to clean them up. 
(alternatively, entire pool can be deleted and recreated instead).

Reply:
If more pending requests get deleted it leaves a large number of stale resources and will be left unwantedly in the pool until the pool is deleted.